### PR TITLE
Update USAGE.md

### PR DESCRIPTION
**Pull request by SerbianSensation**
Updated USAGE.md to reflect the new features and functionality of the application.

Added an images directory to the root of the project, which contains images used in USAGE.md to better clarify how to use tab completion and how it works.
  
Please give it a good read, and let me know if something could be edited or worded better, if something is missing, or something is missing.

### CODE REVIEW

File: USAGE.md [hunk 1]
```diff
@@ -5,14 +5,21 @@
...
+
+	1. Save progress to a file on your computer
+	2. Quit the current program
+	3. Run the desired interface (GUI or CLI)
+	4. Load the file that you saved earlier
+	
 # CLI commands
-	save <name>.json (optional <path>) - Save file to specific path or to current directory if no path is specified
-  	load <path> - Loads a file from an absolute path
+	save <name> (optional <path>) - Save progress to a specific path or to the current directory if no path is specified. Adding .json to the end of the file will save it as a JSON file.
```
Comment (HomelessSkittle) [inline USAGE.md:16]: Save in the CLI has changed a little bit since we now use a File object directly instead of trying to do janky stuff with Path objects. Basically, the 'name' of the file would include the path itself, if they want to save it somewhere else.
For example:
save C:\Users\conor\Desktop\umlfile.json would save umlfile.json to my desktop, and that whole thing is the "name"
But
save umlfile.json also works, and would just save to my current working directory.

Both would save a file called umlfile.json, though. It just changes where it actually gets saved to.


TLDR: (optional <path>) can get removed

File: USAGE.md [hunk 2]
```diff
@@ -5,14 +5,21 @@
 	3. gradle shadowJar
 	4. java -jar ./build/libs/mike-all.jar (for GUI) or java -jar ./build/libs/mike-all.jar (for CLI)
```
Comment (HomelessSkittle) [inline USAGE.md:6]: I think this is supposed to be:

**java -jar ./build/libs/mike-all.jar** (for GUI) or **java -jar ./build/libs/mike-all.jar cli** (for CLI)

It might also be nice to include the bold typeface to make the commands stand out more? Personal opinion though.

**Review by HomelessSkittle — CHANGES_REQUESTED** [review]
Minor typo at the very beginning, and we should remove the "optional path" thing from the CLI save command, but other than that it looks like a good usage guide to me. Good work with those pictures especially.

File: USAGE.md (hunk 2 above)
Comment (SerbianSensation) [reply USAGE.md:6]: Ah yea oops, really good catch

File: USAGE.md (hunk 1 above)
Comment (SerbianSensation) [reply USAGE.md:16]: Ah yea forgot about that. Could I make it like I did for load, where I say <filepath>? (The goal being filepath means it include both the path and the file name)

Comment (SerbianSensation) [reply USAGE.md:16]: Supposed to say < filepath > but I guess that makes it invisible lol

**Review by HomelessSkittle — APPROVED** [review]
Alright, yeah, I think it's all good now.
